luda-editor/luda-editor-client/.gitignore

```diff
@@ -1,6 +1,16 @@
-/target
+# Created by https://www.toptal.com/developers/gitignore/api/rust
+# Edit at https://www.toptal.com/developers/gitignore?templates=rust
+
+### Rust ###
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# These are backup files generated by rustfmt
 **/*.rs.bk
-Cargo.lock
-bin/
-pkg/
```

Comment:
`wasm-pack` will create `luda-editor-client/pkg`
I think it's better to move `pkg` under `target` dir in future.
But for now, I think pkg should be ignored.

Reply:
Ahhh wasm-pack is already doing it. Forget it😊.